auth/authenticator.hh

```diff
@@ -129,6 +134,18 @@ public:
     ///
     virtual future<custom_options> query_custom_options(std::string_view role_name) const = 0;
 
+    ///
+    /// Check if the authenticator uses salted hashes.
```

Comment:
this comment is just function name with _  replaced with spaces. reading it doesn't make me understand what this function is for